src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import Layout from "../components/layout/layout";
+const CodeOfConduct = () => {
```

Comment:
```suggestion
import codeConductImg from '../images/codeconduct.svg'

const CodeOfConduct = () => {
```

Aqui la imagen deberia estar en la carpeta images y deberias importarla de esta manera